configserver/src/main/java/com/yahoo/vespa/config/server/http/v2/ApplicationHandler.java

```diff
@@ -137,8 +138,12 @@ public HttpResponse handleGET(HttpRequest request) {
             return applicationRepository.getLogs(applicationId, hostname, apiParams);
         }
 
-        if (isMetricsRequest(request)) {
```

Comment:
Is there anything that currently uses the old path?

Reply:
Please see the isVintageMetrics(request) method for now.

Reply:
No breakage, assuming old path is not used. Removing isVintageMetrics.